front/src/texts/texts.json

```diff
@@ -168,5 +168,27 @@
     "previousTooltip": ["Edellinen sivu", "Previous page"],
     "firstTooltip": ["Ensimmäinen sivu", "First page"],
     "lastTooltip": ["Viimeinen sivu", "Last page"]
+  },
+  "month": {
+    "0": ["Err", "Err"],
+    "01": ["Tammikuu", "January"],
+    "1": ["Tammikuu", "January"],
+    "02": ["Helmikuu", "February"],
+    "03": ["Maaliskuu", "March"],
+    "04": ["Huhtikuu", "April"],
+    "05": ["Toukokuu", "May"],
+    "06": ["Kesäkuu", "June"],
+    "07": ["Heinäkuu", "July"],
+    "08": ["Elokuu", "August"],
+    "09": ["Syyskuu", "Septemper"],
+    "10": ["Lokakuu", "October"],
+    "11": ["Marraskuu", "November"],
+    "12": ["Joulukuu", "Decemper"],
```

Comment:
Check month name English spelling

Reply:
Should be right. do you see any misspellings?

Reply:
September, December